docs/content/mixin-distribution.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
Yeah I wasn't sure about that file or not? I realized that it was a placeholder that never got filled out and wasn't sure if I should start now or not. 😝 

I was thinking of making that page generated from the cobra help text because otherwise it's doomed to always be out of date. Let me know what you think!

Reply:
I think a generated page in its stead would be great, for the reasons you've mentioned!  👍